src/coreclr/vm/class.cpp

```diff
@@ -1430,6 +1430,9 @@ MethodDesc* MethodTable::GetExistingUnboxedEntryPointMD(MethodDesc *pMD)
 bool MethodTable::IsHFA()
 {
     LIMITED_METHOD_CONTRACT;
+#if defined(TARGET_LOONGARCH64)
```

Comment:
`FEATURE_HFA` seems to be not defined on LoongAarch64 so this whole block should not exists. If HFA not required, maybe better make sure that in `coreclr/inc/switches.h` `FEATURE_HFA` is not defined.

Reply:
> `FEATURE_HFA` seems to be not defined on LoongAarch64 so this whole block should not exists. If HFA not required, maybe better make sure that in `coreclr/inc/switches.h` `FEATURE_HFA` is not defined.

Yes, I will update it.
Thanks for your review~